src/pool/cli.c

```diff
@@ -423,6 +423,8 @@ pool_connect_cp(tse_task_t *task, void *data)
 out:
 	crt_req_decref(arg->rpc);
 	map_bulk_destroy(pci->pci_map_bulk, map_buf);
+	/* Ensure credential memory is wiped clean */
+	memzero_explicit(pci->pci_cred.iov_buf, pci->pci_cred.iov_buf_len);
```

Comment:
Shouldn't we use memset_s() instead since it has been built exactly for this purpose?

Reply:
or explicit_bzero() like in https://www.gnu.org/software/libc/manual/html_node/Erasing-Sensitive-Data.html
We shouldn't have to add any assembly code for that and just use the built-in function.

Reply:
Neither of those are available on Linux. explicit_bzero is only available in the version of libc for OpenBSD.